### PR TITLE
[improvment] show export support label like

### PR DESCRIPTION
**Pull request by wunan1210**
#  Proposed changes

using `show export where label like 'xxx%'` to list more results.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (Yes)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

just copy blindly...  😓

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowExportStmt.java [hunk 1]
```diff
@@ -132,82 +137,47 @@ private void analyzePredicate(Expr whereExpr) throws AnalysisException {
...
-            
-            // right child
-            if (hasState) {
-                if (!(whereExpr.getChild(1) instanceof StringLiteral)) {
-                    valid = false;
-                    break CHECK;
+        boolean valid = false;
+
+        // enumerate all possible conditions
+        if (whereExpr.getChild(0) instanceof SlotRef) {
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowExportStmt.java:143]: Could you please add some test

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java [hunk 2]
```diff
@@ -147,7 +155,9 @@ private ExportJob createJob(long jobId, ExportStmt stmt) throws Exception {
                     continue;
                 }
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java:157]:
```suggestion
If ( label not null or empty) {
    if (isLabelUseLike) {
      check pattern;
    } else {
       check job label;
    }
}
```

File: fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java [hunk 3]
```diff
@@ -147,7 +155,9 @@ private ExportJob createJob(long jobId, ExportStmt stmt) throws Exception {
...
 
-                if (!Strings.isNullOrEmpty(label) && !jobLabel.equals(label)) {
+                if (!Strings.isNullOrEmpty(label) &&
```
Comment (EmmyMiao87) [inline fe/fe-core/src/main/java/org/apache/doris/load/ExportMgr.java:158]: The above way of writing is more readable

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**